.rubocop.yml

```diff
@@ -20,9 +20,22 @@ Style/FileName:
     - '**/*.rake'
     - 'valhalla/Gemfile'
     - 'Capfile'
+    - 'simple-tiles/Vagrantfile'
 Style/PredicateName:
   Exclude:
     - 'app/controllers/catalog_controller.rb'
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - 'simple-tiles/Vagrantfile'
+Style/BarePercentLiterals:
+  Exclude:
+    - 'simple-tiles/Vagrantfile'
+Style/PercentLiteralDelimiters:
+  Exclude:
+    - 'simple-tiles/Vagrantfile'
+Style/UnneededPercentQ:
+  Exclude:
+    - 'simple-tiles/Vagrantfile'
```

Comment:
That simple-tiles directory is an artifact of compiling the simple tiles library on circle. I would have thought that AllCops exclusion on line 15 would have taken care of this. Does rubocop fail without these now? Alternately, we can remove that file as part of the circle configuration process. Maybe add a `rm Vagrantfile` here: https://github.com/pulibrary/figgy/blob/master/bin/ci_simple_tiles_install.sh#L11

Reply:
👍 Can we move the simple-tiles directory to tmp and exclude the entire tmp directory?  It seems like Rubocop shouldn't be checking stuff like this that's not our code.